src/bp/sdk/botpress.d.ts

```diff
@@ -1576,10 +1574,20 @@ declare module 'botpress/sdk' {
       event: IO.OutgoingEvent,
       incomingEvent: IO.IncomingEvent,
       bp: typeof import('botpress/sdk')
-    ): Promise<void>
+    ): Promise<boolean>
+  }
+
+  export interface ExtractionResult {
+    value: any
```

Comment:
You could use generics so the prompt builder can explicitely tell the value type
```
ExtractionResult<T> { value: T }
```